### ISSUE TITLE
A number of Amiga demos have stopped launching [compatibility?]

### ISSUE THREAD

**Issue by giantclambake** [BlitterStudio/amiberry#1276]
I routinely check/test amiberry using the Pimiga (3/4) & ClassicWB Amiga environments ~ I check both using rpi4 & x86-64. 

When I say 'a number of', I could create a list of demos that did start/run previously, but now (sometime since v6.x.x) these titles just 'blink' the screen and return to workbench desktop...ie; failed to launch (a few might hit guru suspend/reboot window).

I did however find one demo (https://demozoo.org/productions/8093/) which consistently throws the same error, in both RTG and not, with JIT enabled/disabled...and even in vanilla workbench3.0/3.2 installs...using amiberry v5.7.0 on rpi4 & v6.3.0 preview on x86-64 ...

![ex](https://imgur.com/GTbBH0L.png)

...I think in an ideal world, whatever's causing the 'texture.chk' to fail here, may also be the cause of other demos now failing.

I definitely know this demo was working back at v5.6.4 on x86-64, as I left a marker on YT (annotated to remind me of what I was using at the time =)  https://www.youtube.com/watch?v=VvQwMEXI5uE

TIA

**Comment by solskogen** [comment]
That demo works fine on my Pi5.

**Comment by giantclambake** [comment]
@solskogen  ~ thanks for checking ; a little bit of my bad for just referencing the demo URL, and not testing that actual archive (I was using copies as shipped with the workbench setups mentioned). The demo I linked to does indeed work fine here.

It would seem something's gone awry somewhere with aforementioned distributions -- the 'eph-fels' directory has another 'eph-fels' nestled under it. and the place is littered with *.uaem files. Strangely, if I roll back to v5.6.6 it all just works, if I go to v6.1.0-preview, only the exe in the nested dir works...neither seem to work anymore in v.6.3.0 on x86-64....

...BUT....the original demo archive works, and seeing this means I'll have to recheck the other not-running demos....

Closing this as someone else's problem, not amiberry's ;)
